src/Tasks/Common/Resources/Strings.resx

```diff
@@ -506,8 +506,11 @@ The following are names of parameters or literal values and should not be transl
     <value>NETSDK1100: Windows is required to build Windows desktop applications.</value>
     <comment>{StrBegin="NETSDK1100: "}</comment>
   </data>
-  <data name="ILLink_Info" xml:space="preserve">
-    <value>Optimizing assemblies for size, which may change the behavior of the app. Be sure to test after publishing. See: https://aka.ms/dotnet-illink</value>
+  <data name="ILLinkRunning" xml:space="preserve">
+    <value>Optimizing assemblies for size. This process might take a while</value>
```

Comment:
I think we should print out something like the original message if trim warnings are suppressed. Reasoning:
* There's no message telling people that trimming is potentially problematic with warnings off.
* It's the default setting for Blazor/Xamarin and probably MAUI as well

I know that info message is not very strong, but I think we should not forgo a chance to "warn" people about potential issues.